pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -2750,7 +2754,7 @@ let
           license = lib.licenses.mit;
         };
       };
-    };
+    });
 
   aliases = self: super: {
     # aliases
```

@schuelermine  Sandro probably means

```suggestion
    # aliases
    _1Password = lib.trace ''_1Password has been replaced with "1Password"'' self."1Password";
    _2gua = lib.trace ''_2gua has been replaced with "2gua"'' self."2gua";
    _4ops = lib.trace ''_4ops has been replaced with "4ops"'' self."4ops";
```

and remove the new alias section at line 33-37.

---

Ah, I see.
I’ve done so now.